src/renderer/components/external-player-settings/external-player-settings.js

```diff
@@ -5,6 +5,7 @@ import FtSelect from '../ft-select/ft-select.vue'
 import FtInput from '../ft-input/ft-input.vue'
 import FtToggleSwitch from '../ft-toggle-switch/ft-toggle-switch.vue'
 import FtFlexBox from '../ft-flex-box/ft-flex-box.vue'
+import i18n from '../../i18n/index.js'
```

Comment:
All Vue components have access to a global `i18n` object without the need to import. Please use that instead. It can be accessed with `this.$t`. Please check some other components for more information on how it's used.